src/Microsoft.VisualStudio.VsInteractiveWindow/VsInteractiveWindowCreationParameters.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.InteractiveWindow.Shell 
+{
+    /// <summary>
+    /// Provides creation parameters to the interactive window factory.
+    /// </summary>
+    public class VsInteractiveWindowCreationParameters 
```

Comment:
If it was useful for the consumer to pass an instance of the class around of the API (like CompilationOptions in Roslyn e.g.) then this design would be better. But I expect the consumers of the API would just create an instance at the call-site of the Create method.

---
In reply to: [91173204](https://github.com/dotnet/interactive-window/pull/23#discussion_r91173204) [](ancestors = 91173204,91171303)